listeph/ratemylandlord
# Add New Landlord Page

### What is in this PR
* created basic form for adding landlord, that sends a POST request to backend on submit
* Here is an image where the POST request's response is console.log'ed ![Screen Shot 2021-11-02 at 4 56 20 PM](https://user-images.githubusercontent.com/43322572/139956892-42ce4fc0-298b-420b-a68d-84b123ebbb59.png)
* added some generated files to `.gitignore`
### How To Test
* The page can be viewed at {localhost url}/add
* the POST request response is still console.log'ed as of now, and you can also check the resulting landlords in the database at {localhost url}/api/home
